modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppStepContent.es.js

```diff
@@ -67,7 +73,35 @@ const EditAppStepContent = ({
 					/>
 				</ClayLayout.ContentCol>
 
-				{shortCutButton}
+				{shortCutButton && (
+					<ClayButton
+						displayType="secondary"
+						onClick={() => {
+							window.open(
+								workflowProcessBuilderPortletURL,
+								'_blank'
+							);
+							setShowInfoAlert(true);
+						}}
+					>
+						{Liferay.Language.get('manage-workflows')}
+						<ClayIcon className="ml-2" symbol="shortcut" />
+					</ClayButton>
+				)}
```

Comment:
@naraandrade is this logic specific to the workflow step? You shouldn't change the shortCutButton previous implementation.

Reply:
@naraandrade what do you think about move the alert display control implementation to outside of the stepContent component like the previous shotCurButton implementation?